Libplanet.Explorer.Executable/Program.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Libplanet.Action;
+using Libplanet.Explorer.Interfaces;
+using Libplanet.Store;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Mono.Options;
+
+namespace Libplanet.Explorer.Executable
+{
+    /// <summary>
+    /// The program entry point to run a web server.
+    /// </summary>
+    public class Program
+    {
+        private static OptionSet options = new OptionSet
+        {
+            {
+                "s|store-type=",
+                "The storage backend to use.  Available types: " +
+                    string.Join(", ", StoreRegistry.List()) + ".",
+                v => storeTypeName = v
+            },
+            {
+                "c|chain-id=",
+                "The chain ID to view.  Omittable if there is only one ID.",
+                v => chainId = v
+            },
+            {
+                "h|help",
+                "Show this message and exit.",
+                v => showHelp = !(v is null)
+            },
+        };
+
+        private static bool showHelp;
+
+        private static string storeTypeName;
+
+        private static string chainId;
+
+        public static int Main(string[] args)
+        {
+            string programName = System.Environment.GetCommandLineArgs()[0];
+            int code = Parse(programName, args);
```

Comment:
`programName` seems to be no longer used in this scope. Can we treat it in `Parse()`?